codersdk/workspaceapps.go

```diff
@@ -62,6 +62,7 @@ type WorkspaceApp struct {
 	Healthcheck Healthcheck        `json:"healthcheck"`
 	Health      WorkspaceAppHealth `json:"health"`
 	Hidden      bool               `json:"hidden"`
+	OpenIn      string             `json:"open_in"`
```

Comment:
Could you expose this as a typed `const` instead (similarly to `WorkspaceAppSharingLevel`)?

This allows the typescript-generated code to use a union type for this field.